components/dashboard/src/SwitchToPAYG.tsx

```diff
@@ -430,7 +430,7 @@ function SwitchToPAYG() {
         return (
             <div className="flex flex-col max-h-screen max-w-2xl mx-auto items-center w-full mt-24">
                 <Heading2>You're now on pay-as-you-go! 🎊</Heading2>
-                <Subheading>You are one step away from using larger workspaces and custom timeouts.</Subheading>
+                <Subheading>New subscriptions are limited to 1000 credits ($9/mo) by default. Change your monthly limit in the billing page if you need more.</Subheading>
```

Comment:
@AlexTugarev how can I show the corresponding currency ($/€) depending on the user/county?

Reply:
try with, that's what we guesstimate depending on the client's region.  
```
const { currency } = useContext(PaymentContext);
```